### ISSUE TITLE
Board Editor: easier way to switch between layers

### ISSUE THREAD

**Issue by glcerone** [LibrePCB/LibrePCB#915]
##### SUMMARY

Up to now there is a dock widget allowing to select one or more layer to visualize. The layers dock widget allows to easily select on or more layers to show but results, in my opinion, a little bit cumbersome when you want to quickly switch between one layer to another during e.g. trace routing.

##### SOLUTION

It could be useful to add a shortcut allowing to select between "one layer mode" (i.e. only one layer is showed) and "mult-layers mode" (all the layers selected in the dock widget are showed). Altium Designer has SHIFT+F shortcut allowing to use this feature.

Instead of the layers dock widget (or in addition to it) it could be useful to have a bottom toolbar in the pcb editor showing as many buttons as the layers that is possible to show.

When you are in "one layer mode", you should be able to quickly select which layer to show by simply pressing one of the button in the toolbar.

Again, in Altium Designer it is possible to switch between visible layers by simply pressing the "+" (the layer after the current layer is showed) and "-" (the layer before the current layer is showed) shortcuts.

The image below shows and example of the "layers bar". I could help implementing this feature if possible and useful.

<img width="617" alt="layersbar_example" src="https://user-images.githubusercontent.com/21297378/133907287-5e8414f6-53c6-4785-81f7-c94079987fe6.PNG">

**Comment by EdizonTN** [comment]
And please, don't forget control this feature by user configurable hot-keys also.

**Comment by SiboVG** [comment]
This is an important request to me. When I first opened a board in LibrePCB, I was really confused about how to switch layers. I prefer something that's always visible, like the layers bar from Altium.

**Comment by ubruhin** [comment]
I agree. I already have some ideas in mind, I hope to get it implemented in v2.2.0.
